pom.xml

```diff
@@ -218,7 +221,7 @@
         <version.org.syslog4j>0.9.30</version.org.syslog4j>
         <version.org.wildfly.arquillian>2.1.0.Final</version.org.wildfly.arquillian>
         <version.org.wildfly.bridge.servlet-api-bridge>1.0.0.Final</version.org.wildfly.bridge.servlet-api-bridge>
-        <version.org.wildfly.build-tools>1.2.6.Final</version.org.wildfly.build-tools>
+        <version.org.wildfly.build-tools>1.2.8.Final</version.org.wildfly.build-tools>
```

Comment:
I don't think we'll be able to do this as it changes the order of the maven coordinates.